es5.md

```diff
@@ -531,7 +531,7 @@ A unary operator token.
 interface UpdateExpression <: Expression {
     type: "UpdateExpression";
     operator: UpdateOperator;
-    argument: Expression;
+    argument: Pattern | CallExpression;
```

Comment:
No, see discussion in previous PRs. In ES5 specifications **must** reject as an early error any LHS that can't evaluate to `Reference` in runtime, and only `CallExpression` and `NewExpression` are such that could result in `Reference` despite being expressions. For literals, unary, binary, conditional and sequence expressions there is no way in spec they can evaluate to `Reference` and thus must be rejected in such position.